examples/http-to-websocket-upgrade/http-to-websocket-upgrade.bal

```diff
@@ -36,27 +32,40 @@ service<http:Service> httpService bind servicEp {
 
         _ = conn -> respond(resp);
     }
+
+
+    @http:ResourceConfig {
+        webSocketUpgrade:{
+            upgradePath:"/ws",
+            upgradeService:wsService
+        }
+    }
+    upgrader(endpoint ep, http:Request req) {
+    }
 }
 
-@Description {value:"When a WebSocket upgrade path is defined under HTTP in the WebSocket configuration, there can be \n - Full service configuration: Two base paths for the same service from either HTTP or WebSocket. \n - Without service configuration: WebSocket service is a slave service of the HTTP service. Only the upgrade path can be there. \n - Configuration without basePath: Acts as a slave service but can configure sub protocols, idle timeout, etc."}
+
+//Note: When a WebSocket upgrade path is defined in HTTP resource configuration
+//- Without service configuration for WebSocketService default values are taken for sub protocols, idle timeout etc...
+//- If  WebSocketServiceConfig is defined without the path, sub protocols, idle timeout etc... can be configured
+//- If path is defined in the WebSocketServiceConfig it shall be ignored
+//- This service can also be bound to a different endpoint in which case the path configuration will become useful
 @http:WebSocketServiceConfig {
     subProtocols:["xml, json"],
-    idleTimeoutInSeconds:5
+    idleTimeoutInSeconds:20
 }
 service<http:WebSocketService> wsService {
 
-    onOpen (endpoint ep) {
-        var conn = ep.getClient();
+    onOpen(endpoint ep) {
         io:println("New WebSocket connection: " + ep.id);
     }
 
-    onText (endpoint ep, string text) {
+    onText(endpoint ep, string text) {
         io:println(text);
         _ = ep -> pushText(text);
```

Comment:
Since we should enforce the best practices shouldn't we add "but" operation and see if there is an error here 
https://github.com/ballerina-platform/ballerina-examples/blob/6381caf3ed2c27ce5a831ec5f417055fde5c4bc5/examples/websocket-proxy-server/websocket-proxy-server.bal#L38